src/gui/editors/PianoRoll.cpp

```diff
@@ -2646,7 +2646,7 @@ void PianoRoll::mouseMoveEvent( QMouseEvent * me )
 					)
 				{
 					// delete this note
-					m_midiClip->removeNote( note );
+					it = m_midiClip->removeNote(note);
```

Comment:
```suggestion
					it = m_midiClip->removeNote(it);
```
You can use the new function like this